src/frontend/src/components/SearchPage/SearchPage.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useEffect } from 'react';
+import { useLazyQuery } from '@apollo/react-hooks';
+import gql from 'graphql-tag';
+import Header from '../Header';
+import SearchBar from '../SearchBar';
+import AuthorResult from '../AuthorResult';
+
+const SearchPage = () => {
+  const SEARCH_QUERY = gql`
+    query SearchAuthorQuery($author: String!) {
+      getPosts(page: 0, perPage: 25, filter: { author: $author }) {
+        title
+        published
+        url
+        feed {
+          id
+          author
+        }
+      }
+    }
+  `;
+
+  const [searchText, setSearchText] = useState('');
+  const [results, setResults] = useState([]);
+
+  const [executeSearch, { data }] = useLazyQuery(SEARCH_QUERY, {
+    // setting author variable here to use in the query above
+    variables: { author: searchText },
+    onCompleted: () => {
+      const returnedIds = [];
+      // We only want to display an author once, this would avoid the issue if
+      // there are multiple posts by the same author returned in our query
+      const finalResults = data.getPosts
+        .map(result => {
+          if (!returnedIds.includes(result.feed.id)) {
+            returnedIds.push(result.feed.id);
+            return {
+              id: result.feed.id,
+              author: result.feed.author,
+              // The post will contain information about their most recent post to be used for AuthorResult component
+              post: {
+                title: result.title,
+                postLink: result.url,
+                postDate: result.published,
+              },
+            };
+          }
+        })
+        // Filters undefined items that might have been pushed/mapped when we're checking for duplicate author results
+        .filter(item => item);
+      setResults(finalResults);
+    },
+  });
+
+  // Hook that will re-render the page only if the state of results changes
+  useEffect(() => {}, [results]);
+
+  function onClickHandler() {
+    executeSearch();
+  }
+
+  function onChangeHandler(event) {
+    setSearchText(event.target.value);
+  }
+
+  return (
+    <div>
+      <Header />
+      <div
+        style={{
+          height: '12vh',
+        }}
```

Comment:
For good practice, lets define a min-height as well if we are using `vh` units since this could cause scaling issue.

```
minHeight: '12em`
```

For example will leverage 120px min height if that's what we are going for. Adjust to taste